### ISSUE TITLE
Swagger::Blocks::DeclarationError (swagger_root must be declared)

### ISSUE THREAD

**Issue by jun1st** [fotinakis/swagger-blocks#93]
I encounter an error as stated in the title, `Swagger::Blocks::DeclarationError (swagger_root must be declared)`

but I have added this `swagger_root` in my `apidocs_controller.rb` file

```
class ApidocsController < ActionController::Base
  include Swagger::Blocks

  swagger_root do
    key :swagger, '2.0'
    info do
      key :version, '1.0.0'
      key :title, 'Swagger Petstore'
      key :description, 'A sample API that uses a petstore as an example to ' \
                        'demonstrate features in the swagger-2.0 specification'
      key :termsOfService, 'http://helloreverb.com/terms/'
      contact do
        key :name, 'Wordnik API Team'
      end
      license do
        key :name, 'MIT'
      end
    end
    tag do
      key :name, 'pet'
      key :description, 'Pets operations'
      externalDocs do
        key :description, 'Find more info here'
        key :url, 'https://swagger.io'
      end
    end
    key :host, 'petstore.swagger.wordnik.com'
    key :basePath, '/api'
    key :consumes, ['application/json']
    key :produces, ['application/json']
  end

  SWAGGERED_CLASSES = [
    Api::V1::PostsController
  ].freeze

  def index
    render json: Swagger::Blocks.build_root_json(SWAGGERED_CLASSES)
  end
end
```

what did I miss here?

**Comment by davemcg3** [comment]
I just worked through the same thing.  Try adding ", self" to your SWAGGERED_CLASSES array because you didn't include your Apidocs controller.

SWAGGERED_CLASSES = [
    Api::V1::PostsController,
    self,
  ].freeze

**Comment by fotinakis** [comment]
Fixed by #94
